backend/coins/coin/conversions.go

```diff
@@ -64,7 +64,15 @@ func Conversions(amount Amount, coin Coin, isFee bool, ratesUpdater *ratesPkg.Ra
 }
 
 // ConversionsAtTime handles fiat conversions at a specific time.
-func ConversionsAtTime(amount Amount, coin Coin, isFee bool, ratesUpdater *ratesPkg.RateUpdater, formatBtcAsSats bool, timeStamp *time.Time) map[string]string {
+// It returns the map of conversions and a bool indicating if the rates have been estimated
+// using the latest instead of the historical rates for recent transactions.
+func ConversionsAtTime(amount Amount, coin Coin, isFee bool, ratesUpdater *ratesPkg.RateUpdater, formatBtcAsSats bool, timeStamp *time.Time) (map[string]string, bool) {
+	latestRatesTime := ratesUpdater.LatestHistoricalUpdate(string(coin.Code()))
```

Comment:
shouldn't this be done fiat by fiat? If a user has for example CHF and EUR as fiats, and EUR is up to date but CHF is not, then for EUR it should use the historical value even if for CHF it is not available.

Reply:
I think that making this at the level of each fiat would over-complicate: I'd prefer not to change too much the Amount structure as it is used a lot across both backend and frontend. Also, I don't think we should expect big differences in historical rates timestamps of different currencies right? 
I kept this approach for now, but I can still change to a single fiat level, if you're not convinced.